docs/toolbox/accessibility_indicators/oev_gueteklassen.md

```diff
@@ -11,10 +11,19 @@ The Public Transport Quality Classes <i>(German: ÖV-Güteklassen)</i> show the
 
 ## 1. Explanation
 
-Public Transport Quality Classes, also known as **ÖV-Güteklassen**, are a classification system used to evaluate and categorise the **quality of public transport** services in a given area. The concept is used to plan and evaluate public transport services to ensure that they meet certain standards and effectively serve the needs of the population. The **quality classes** thereby range from **<span style={{color: "#199741"}}>A</span>** (very good offer) to **<span style={{color: "#E4696A"}}>F</span>** (very poor offer).
+Public Transport Quality Classes, also known as **ÖV-Güteklassen**, are a classification system used to evaluate and categorize the **quality** of public transport** services in a given area. The concept is used to plan and evaluate public transport services to ensure that they meet certain standards and effectively serve the needs of the population. The **quality classes** thereby range from **<span style={{color: "#199741"}}>A</span>** (very good offer) to **<span style={{color: "#E4696A"}}>F</span>** (very poor offer).
 
 ![ÖV-Güteklassen in GOAT](/img/toolbox/accessibility_indicators/gueteklassen/example.png "ÖV-Güteklassen in GOAT")
 
+:::info 
+The calculation of the public transport quality classes is only available for areas where the transport network of the selected routing mode is integrated into GOAT. As soon as you select a routing mode, you will see the **geofence** in which the calculation is possible.
```

Comment:
Similar to the previous comment, this indicator depends on the availability of PT/GTFS data and not a user-selected routing mode.